### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by xuwei-k**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -8,13 +8,12 @@ import scala.language.postfixOps
 
 lazy val currentVersion = "2.1.0"
```
Comment (seratch) [inline build.sbt]: The next release with this change would be 2.2.0 (as it contains jackson minor upgrades)

File: micro/src/test/scala/example/TypedHelloSpec.scala [hunk 2]
```diff
@@ -82,7 +82,7 @@ class TypedHelloSpec extends ScalatraFlatSpec {
 
   it should "return cookies in the ActionResult" in {
     get("/cookie") {
-      header("Set-Cookie") should equal("foo=bar;Path=/")
+      header("Set-Cookie") should equal("foo=bar; Path=/")
```
Comment (seratch) [inline micro/src/test/scala/example/TypedHelloSpec.scala]: Hahm, has Jetty changed the header value (probably again), right?

**Review by seratch — APPROVED** [review]
Wonderful PR! Once the CI build passes, I'll merge this.

File: micro/src/test/scala/example/TypedHelloSpec.scala (hunk 2 above)
Comment (xuwei-k) [reply micro/src/test/scala/example/TypedHelloSpec.scala]: Yes

File: build.sbt (hunk 1 above)
Comment (xuwei-k) [reply build.sbt]: done

**Pull request merged**